target/linux/ramips/dts/mt7621_mikrotik.dtsi

```diff
@@ -27,6 +27,8 @@
 	flash@0 {
 		compatible = "jedec,spi-nor";
 		reg = <0>;
+		#address-cells = <1>;
+		#size-cells = <1>;
```

Comment:
It's not a correct fix. ref: https://lore.kernel.org/all/CAL_JsqLCVjbsPfNOOqfsZQY-KUp8foJqwjK54JU+YZOfonkS7w@mail.gmail.com/